app/src/main/java/io/apicurio/registry/storage/RegistryStorage.java

```diff
@@ -37,12 +38,12 @@
     public Map<String, String> createArtifact(String artifactId, String content) throws ArtifactAlreadyExistsException, RegistryStorageException;
     
     /**
-     * Deletes an artifact by its unique id.
+     * Deletes an artifact by its unique id. Returns list of artifacts versions.
      * @param artifactId
      * @throws ArtifactNotFoundException
      * @throws RegistryStorageException
      */
-    public void deleteArtifact(String artifactId) throws ArtifactNotFoundException, RegistryStorageException;
+    public Set<Long> deleteArtifact(String artifactId) throws ArtifactNotFoundException, RegistryStorageException;
```

Comment:
I'm fine with this, but am not sure where it is needed.  Does the confluent API return a list of version #s when deleting a subject?

Reply:
>> Does the confluent API return a list of version #s when deleting a subject?
Yes.

`      deletedVersions = schemaRegistry.deleteSubjectOrForward(headerProperties, subject);
    } catch (SchemaRegistryException e) {
      throw Errors.schemaRegistryException("Error while deleting the subject " + subject,
                                           e);
    }
    asyncResponse.resume(deletedVersions);
`